pkgs/by-name/to/toml-sort/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+let
+  version = "0.23.1";
+in
+python3Packages.buildPythonPackage {
+  pname = "toml-sort";
+  inherit version;
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "pappasam";
+    repo = "toml-sort";
+    rev = "v${version}";
+    hash = "sha256-7V2WBZYAdsA4Tiy9/2UPOcThSNE3ZXM713j57KDCegk=";
+  };
+
+  nativeBuildInputs = [ python3Packages.poetry-core ];
+
+  propagatedBuildInputs = [ python3Packages.tomlkit ];
```

```suggestion
  dependencies = [ python3Packages.tomlkit ];
```
To conform `pyproject.toml`